src/test/java/exploratory/decodes/JvmTest.java

```diff
@@ -1,6 +1,5 @@
-package decodes.test;
+package exploratory.decodes;
 
-import java.io.IOException;
 import java.util.ArrayList;
 
 public class JvmTest
```

Comment:
This looks like an exploratory test for investigating the Java `Process` API. There are no tests or OpenDCS domain logic within this class. This looks like a candidate for deletion unless someone thinks it is useful to keep around.

Reply:
I say delete those. If it's testing "does the JVM work" that should have a good justification if we're going to keep it. kinda made since when java was new but these days it's pretty stable.

Reply:
Agreed